py/desitarget/cuts.py

```diff
@@ -237,7 +239,7 @@ def isGAIA_STD(ra=None, dec=None, galb=None, gaiaaen=None, pmra=None, pmdec=None
                       gaiagmag=gaiagmag, gaiabmag=gaiabmag, gaiarmag=gaiarmag)
 
     # ADM restrict to point sources.
-    ispsf = gaia_psflike(gaiaaen, gaiagmag)
+    ispsf = gaia_psflike(gaiaaen, gaiagmag, dr=dr)
```

Comment:
Consider reverting all changes in `isGAIA_STD()`.

Reply:
Reverted.